### ISSUE TITLE
Items cannot be centered on same row with fit

### ISSUE THREAD

**Issue by garretto** [obartra/reflex#226]
```javascript
import React from 'react'
import Button from '../components/Button'
import { Grid as G, Col } from 'xn-reflex'

const s = {}

s.container = {
  height: '48px',
  borderRadius: '2px',
  backgroundColor: '#4B4F57',
  boxShadow: '0 0 4px 0 #1D1F21',
  flexFlow: 'row',
}

export default class AlertNotification extends React.Component {
  render() {
    let { alerts } = this.props

    if (alerts.length < 1) return null

    if (alerts.length === 1) {
      let alert = alerts[0]
      return (
        <G style={s.container} size="fit" align="center">
          <Col margin={[1, 0.5, 1, 0.75]} size="auto">{alert.title}</Col>
          <Col margin={[1 / 3]} size="fit"><Button size="small" onClick={alert.action}>{alert.actionText}</Button></Col>
        </G>
      )
    }

    // Return alert list
    return <div />
  }
}
```

<img width="720" alt="screen shot 2017-08-18 at 3 27 06 pm" src="https://user-images.githubusercontent.com/784852/29479921-bc5dae46-8429-11e7-8095-57803c004b93.png">

**Comment by garretto** [comment]
Had to add ` flexFlow: 'row',` to the style

**Comment by obartra** [comment]
I added a test case in #228 but it already seems to behave correctly. I wonder if we serendipitously fixed it as part of the width changes on IE11.

If you upgrade to `xn-reflex` version 10, does the issue persist?

**Comment by obartra** [comment]
I just merged #228 changes, feel free to reopen if the issue persists
